client/actor.go

```diff
@@ -22,10 +22,10 @@ import (
 	anypb "github.com/golang/protobuf/ptypes/any"
 	"github.com/pkg/errors"
 
-	pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
 	"github.com/dapr/go-sdk/actor"
 	"github.com/dapr/go-sdk/actor/codec"
 	"github.com/dapr/go-sdk/actor/config"
+	pb "github.com/dapr/go-sdk/dapr/proto/runtime/v1"
```

Comment:
Why not use `dapr/dapr`'s proto package?

Reply:
It is too heavy for dapr `go-sdk` to directly depend on the `dapr/dapr` repo. I suggest that it is best to separate proto into the standard repo that named `protocol`.

At present,  go-sdk directly imports the `dapr/dapr` module, which will cause the micrologic service to import `dapr/dapr` in go.mod. It is appropriate for the business side to only introduce APIs.

1. Related issue: https://github.com/dapr/dapr/issues/4920
2. Related PR: https://github.com/dapr/dapr/pull/4923

Reply:
1. 
https://github.com/dapr/go-sdk/tree/main/dapr/proto

2. 
![image](https://user-images.githubusercontent.com/20457624/190980018-e6aca182-3b41-43f4-9341-3d8f84011683.png)

In the previous go-sdk, the introduction of dapr proto was confusing. It not only directly imported `dapr/dapr` proto, but also directly copied proto to `github.com/dapr/go-sdk/dapr/proto`.

So i removed `dapr/dapr` dependency and copy proto to this repo.